### PR TITLE
Editorial: fix trivial grammar error

### PR DESCRIPTION
**Pull request by xiaoyinl**
Fix a run-on sentence in Section 20.1.3.6.

### CODE REVIEW

**Comment by ConnerAiken** [comment]
Why not, "... dependent. However, xyz."

**Comment by xiaoyinl** [comment]
@ConnerAiken I made the change you suggested. Thanks!
@ljharb Thanks for reviewing! If I remember correctly, "however" can't be used as a conjunction in this way in U.S. English, so I think it's better to change the punctuation.

**Comment by ljharb** [comment]
@xiaoyinl http://www.quickanddirtytips.com/education/grammar/starting-a-sentence-with-however-right-or-wrong

**Comment by ConnerAiken** [comment]
Exactly. "everyone knows it means 'nevertheless.'" .

**Comment by littledan** [comment]
@ljharb The link you provided seems to say that it's fine to start a sentence with 'however'. I usually prefer the style in this PR. Long sentences, generally, can be hard to follow.

**Comment by ljharb** [comment]
Totally, the link tells me that both styles are acceptable. I have no aesthetic preference for either.

**Comment by bterlson** [comment]
"However" in this case is a conjunctive adverb and thus the sentence is not a run-on technically speaking, however (😀) whether it's too long is up to stylistic choice. I find the existing wording more clear personally. It's obvious "however" is qualifying the previous part of the sentence (with the PR, the however leaves me wondering, what are you howevering exactly?). Also, using However at the beginning of the sentence introduces an ambiguity about the definition of however until you reach the comma. Lastly, while I am more of a descriptivist, I recall style guides suggesting that starting a sentence with However when you mean "nevertheless".

@xiaoyinl thank you for raising this issue, and sorry for closing it. Please feel free to continue suggesting wording updates though!
